docs/src/components/copy-command-button.tsx

```diff
@@ -0,0 +1,68 @@
+'use client';
+
+import { useState } from 'react';
+import { copyTextToClipboard } from '../utils';
+import Button from './button';
+
+export function CopyCommandButton() {
+  const copyCommandText = 'npm install @keystatic';
```

Comment:
This should be `npm create @keystatic`

Reply:
Actually make it `npm create @keystatic@latest`

Reply:
Ahh can't believe I typed that!